File: src/C/1008.c
/*
 * 1008 - Salary
 * Author: Otavio
 * Date: 25/04/2024
 */
#include <stdio.h>
 
int
main()
{
    int emp_id, hours;
    double us_per_hour;
    
    scanf("%d", &emp_id);
    scanf("%d", &hours);
    scanf("%lf", &us_per_hour);
    
    double salary = hours*us_per_hour;
    printf("NUMBER = %d\nSALARY = U$ %.2lf\n", emp_id, salary);
    return 0;
}

File: src/JS/1008.js
/*
 * 1008 - Salary
 * Author: Otavio
 * Date: 31/08/2024
 */
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const idNumber = parseInt(lines.shift());
const hours = parseInt(lines.shift());
const amountPerHour = parseFloat(lines.shift());

const salary = hours*amountPerHour;

console.log(`NUMBER = ${idNumber}
SALARY = U$ ${salary.toFixed(2)}`);
